src/components/CommentCard.jsx

```diff
@@ -1,14 +1,33 @@
-import React from 'react'
+import React from "react";
+import { deleteComment } from "../utils/utils";
+import { UserLoginContext } from "../context/UserLoginContext";
+
+const CommentCard = ({
+  comment_id,
+  body,
+  author,
+  created_at,
+  votes
+}) => {
+const { currentUser } = React.useContext(UserLoginContext);
+
+const handleDelete = (event) => {
+  if(currentUser===author)
+  deleteComment(event.target.value)
```

Comment:
Take a look at some optimistic rendering for this please! I'd also recommend removing the delete button if the comment does not belong to the logged in user. Otherwise, it's confusing to have a button there that the user cannot click.